profiles/lib/destination.lua

```diff
@@ -24,4 +28,48 @@ function Destination.get_destination(way)
   return rv
 end
 
+
+function Destination.get_destination_forward(way)
+  local destination_ref = way:get_value_by_key("destination:ref")
+  local destination_forward = way:get_value_by_key("destination:forward")
```

Comment:
Just to throw a wrench in the works here:

http://taginfo.openstreetmap.org/keys/destination%3Aref%3Aforward

`destination:ref` can *also* have forward/backward versions :-P  They're not super common, but there are several thousand of them out there in the wild, so we might as well add support for them while we're at it.